config/config.go

```diff
@@ -176,7 +176,6 @@ type Config struct {
 	Security                   Security                `toml:"security" json:"security"`
 	Status                     Status                  `toml:"status" json:"status"`
 	Performance                Performance             `toml:"performance" json:"performance"`
-	PreparedPlanCache          PreparedPlanCache       `toml:"prepared-plan-cache" json:"prepared-plan-cache"`
```

Comment:
Should we remove this deprecated item directly or leave it with some comments? @morgo @CbcWestwolf

Reply:
I'm afraid it can cause some failures when decoding an old config file with `prepared-plan-cache` if we remove it directly.